static/src/js/components/SpatialSelection/SpatialSelection.js

```diff
@@ -559,6 +578,27 @@ class SpatialSelection extends Component {
     }
   }
 
+  renderCircle(points, featureGroup) {
+    if (featureGroup) {
+      const center = new L.LatLng(points[1], points[0])
```

Comment:
Prefer destructuring
```suggestion
      const [
        lat,
        lng,
        radius
      ] = points
      const center = new L.LatLng(lng, lat)
```